.travis.yml

```diff
@@ -2,10 +2,12 @@ sudo: false
 
 language: go
 
+env:
+  global:
+  - GO15VENDOREXPERIMENT=1
```

Comment:
Shouldn't we be vendoring at the contrib/ level, not at subdirs?  I
*reallllly* don't want subdirs of contrib/ vendoring other subdirs of
contrib/.

On Tue, May 10, 2016 at 10:31 AM, Eric Paris <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/kubernetes/contrib/pull/943#discussion_r62715889>:
>
> > @@ -2,10 +2,12 @@ sudo: false
> >
> >  language: go
> >
> > +env:
> > +  global:
> > +  - GO15VENDOREXPERIMENT=1
>
> now we don't need this part... doesn't matter.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/contrib/pull/943/files/c77292269bc9ab2fb323be2865a4790f91eacfa8#r62715889>
>